assets/i18n/en.toml

```diff
@@ -4,26 +4,26 @@ docs = "Documentation"
 blog = "Blog"
 github = "GitHub"
 login = "Vehicle logins"
-about = "About evcc"
+about = "About"
 
 [header.theme]
 auto = "Design: system"
 light = "Design: light"
 dark = "Design: dark"
 
 [footer.version]
-availableShort = "update"
-availableLong = "update available"
-modalTitle = "Update available"
-modalUpdateStarted = "Evcc will restart after the update.."
-modalInstalledVersion = "Currently installed version"
-modalNoReleaseNotes = "No release notes available. More information about the new version can be found here:"
+availableShort = "install"
+availableLong = "new version available"
+modalTitle = "New version available"
+modalUpdateStarted = "Restarting after installing the new version…"
```

Comment:
Is `Restarting` really correct here for something that has not yet begun?

Reply:
The app is restarting, but as much is a bit unclear.